docs/_docs/plugins.md

```diff
@@ -59,14 +60,16 @@ values of the gem names of the plugins you'd like to use. An example:
   </p>
 </div>
 
-In general, plugins you make will fall into one of five categories:
+In general, plugins you make will fall broadly into one of five categories:
 
 1. [Generators](#generators)
 2. [Converters](#converters)
 3. [Commands](#commands)
 4. [Tags](#tags)
 5. [Hooks](#hooks)
 
+See the bottom of the page for available examples of each.
```

Comment:
What I'm hoping for is some indication to the reader of what can be found on this page. Perhaps not this solution, but I think it's a good idea to way to somehow inform the reader, near the top of the page before going deep, about the contents of the page, especially so far below the fold. [Alternatively...](https://github.com/jekyll/jekyll/issues/6312#issuecomment-322353194).